Real Analysis/real-analysis-notes.tex

```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+  to $\alpha$ given by \[
+    \pi_\alpha: X \to X_\alpha 
+  \] \[
+    (x_\beta)_{\beta \in A} \mapsto x_\alpha.
+  \]
+  This gives a $\sigma$-algebra on $X$ called the \emph{product
+    $\sigma$-algebra}, denoted \[
+    \bigotimes_{\alpha \in A} \M_\alpha.
+  \]
+  In more concise terms, we say that \[
+    \bigotimes_{\alpha \in A} \M_\alpha = \M(\Ep), \ \Ep =
+    \{\pi_{\alpha}^{-1}(E_\alpha) \mid \alpha \in A, E_\alpha \in
+    \M_\alpha \}.
+  \]
+  Also, to clear up any ambiguity, we note that \[
+    \pi_\alpha^{-1}(E_\alpha) = E_\alpha \times \prod_{\beta \in A
+      \setminus \{\alpha\}} X_\alpha.
+  \]
+\end{example}
+\begin{example}
+  If $A = \{1, \ldots, n\}$ and $X = X_1 \times \cdots \times X_n$,
+  then \[
+    \bigotimes_{\alpha \in A} \M_\alpha = \M_1 \otimes \cdots \otimes \M_n
+  \]
+\end{example}
+\begin{prop}
+  If $A$ is at most countable, then $\bigotimes_{\alpha \in A}
+  \M_\alpha$ is generated by $\prod_{\alpha \in A} E_\alpha$ where
+  $E_\alpha \subset \M_\alpha$. 
+\end{prop}
+\begin{proof}
+  Let \[
+    \Ep_1 = \{\pi_\alpha^{-1}(E_\alpha) \mid \alpha \in A, E_\alpha
+    \in \M_\alpha\}
+  \]
+  and \[
+    \Ep_2 = \{\prod_{\alpha \in A} E_\alpha \mid (E_\alpha)_{\alpha
+      \in A} \text{ family of set such that } \forall \alpha \in A,
+    E_\alpha \in \M_\alpha \}.
+  \]
+  By definition, \[
+    \bigotimes_{\alpha \in A} \M_\alpha = \M(\Ep_1)
+  \]
+  and we seek to show $\M(\Ep_1) = \M(\Ep_2)$.
+  \begin{itemize}
+  \item ($\subset$) It is enough to show $\Ep_1 \subset \Ep_2 \subset
+    \M(\Ep_2)$. Let $\alpha \in A$, $E_\alpha \in \M_\alpha$. Then \[
+      \pi_\alpha^{-1}(E_\alpha) = \prod_{\beta \in A} F_\beta
+    \]
+    where $F_\beta = E_\alpha$ if $\beta = \alpha$ and $F_\beta =
+    X_\alpha$ if $\beta \neq \alpha$. Thus, $\Ep_1 \subset \Ep_2$
+
+  \item ($\supset$) It is enough to show that $\Ep_2 \subset
+    \M(\Ep_1)$. Let $Y = \prod_{\alpha \in A} E_\alpha \in \Ep_2$
+    where, for all $\alpha \in A$, $E_\alpha \in \M_\alpha$. Because
+    $A$ is at most countable, we have that \[
+      Y = \bigcap_{\alpha \in A} \pi_\alpha^{-1}(E_\alpha).
+    \]
+    So, if $x = (x_\alpha)_{\alpha \in Y}$, then $x \in
+    \pi_\alpha^{-1}(E_\alpha) (\in \Ep_1) \ \iff \ x_\alpha \in
+    E_\alpha$. Thus, $Y \in \M(\Ep_1)$ because $Y$ is (at most) countable
+    intersection of elements of $\M(\Ep_1)$ which is a $\sigma$-algebra.
+  \end{itemize}
+\end{proof}
+\begin{prop}
+  Suppose we have $(x_\alpha)_{\alpha \in A}$ and $\Ep_\alpha \subset
+  \P(X_\alpha)$. Let $\M_\alpha = \M(\Ep_\alpha)$ in $X_\alpha$. Then
+  \begin{itemize}
+  \item $\bigoplus_{\alpha \in A} \M_\alpha = \M(\F_1)$ where $\F_1 =
+    \{\pi_\alpha^{-1}(E_\alpha) \mid \alpha \in X, E_\alpha \in \Ep_\alpha\}$
+  \item If, in addition, $A$ is at most countable, then \[
+      \bigoplus_{\alpha \in A} = \M(\F_2)
+    \]
+    with $\F_2 = \{\prod_{\alpha \in A} E_\alpha \mid \forall \alpha
+    \in A, E_\alpha \in \Ep_\alpha\}$
```

Comment:
I don't think "for all alpha" is even sensical.  Haven't we taken the product over all alpha?  So the resulting product is no longer indexed by alpha.  So there is simply no alpha to refer to.